test/acceptance/features/steps/openshift.py

```diff
@@ -300,6 +305,18 @@ def get_resource_info_by_jsonpath(self, resource_type, name, namespace, json_pat
             print(f'Error getting value for {resource_type}/{name} in {namespace} path={json_path}: {output}')
             return None
 
+    def get_json_resource(self, resource_type, name, namespace):
+        error_msg = f"Error in getting resource: '{resource_type}' '{name}' namespace: '{namespace}'"
+        output, exit_code = self.cmd.run(f'{ctx.cli} get {resource_type} {name} -n {namespace} -o json')
+        assert exit_code == 0, error_msg
+        json_output = None
+        try:
+            json_output = json.loads(output)
+        except JSONDecodeError:
```

Comment:
we do not need to catch `JSONDecodeError` to throw another exception here. It is enough to do:

```python
json_output = json.loads(output)
assert json_output is not None, "Error parsing as JSON"
return json_output
```